packaging/zip/ignite3db

```diff
@@ -16,6 +16,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+set -o errexit # Exit on error
+set -o nounset # Exit on unset variables
+set -o pipefail # Exit if any command in a pipeline fails
```

Comment:
A situation with `pipefail` in non-bash shells is [controversal](https://unix.stackexchange.com/questions/654885/who-is-responsible-for-providing-set-o-pipefail/654932). Some shells, especially in legacy environments (even POSIX `sh`), may not support it. I'm afraid it may be risky to use `/bin/sh` as a shebang with it.

Probably, a `/bin/bash` or `/usr/bin/env bash` shebang is more preferrable now.